spras/domino.py

```diff
@@ -205,8 +205,10 @@ def parse_output(raw_pathway_file, standardized_pathway_file):
             edges_df['source'] = edges_df['source'].apply(post_domino_id_transform)
             edges_df['target'] = edges_df['target'].apply(post_domino_id_transform)
             edges_df = reinsert_direction_col_undirected(edges_df)
-
-        edges_df.to_csv(standardized_pathway_file, sep='\t', header=False, index=False)
+            edges_df.columns = ['Node1', 'Node2', 'Rank', 'Direction']
+            edges_df.to_csv(standardized_pathway_file, sep='\t', header=True, index=False)
+        else:
+            edges_df.to_csv(standardized_pathway_file, sep='\t', header=None, index=False)
```

Comment:
Will this output a completely blank file? Should we output the header row now instead? We could do that without pandas if it is hard to do with pandas.

Reply:
Randomly the output file for domino (and for only domino) will add a '\n' to the file when header = True. This was our version of a quick fix to allow for that to not happen.